ui/StatusQ/src/StatusQ/Components/StatusRoundedImage.qml

```diff
@@ -23,6 +23,7 @@ StatusRoundedComponent {
     isLoading: image.isLoading
     isError: image.isError
     border.width: 0
+    radius: 10
```

Comment:
Hmm, this change will affect all other components/views. Better set it at the place where you actually need it

Reply:
Oops, my bad, I got tunnel vision. I'll move it higher in the popup